framework/generated/vulkan_generators/base_generator.py

```diff
@@ -565,7 +569,7 @@ def genGroup(self, groupinfo, group_name, alias):
             enumerants = dict()
             for elem in groupinfo.elem:
                 supported = elem.get('supported')
-                if not supported or not 'disabled' in supported:
+                if not supported or not supported in _remove_supported:
```

Comment:
For some of these, `supported` is a string with multiple comma-separated values.  For example, `VK_KHR_display` has `supported="vulkan,vulkansc"`.

This works for now because (I think) the only values are `disabled`, `vulkan`,`vulkan,vulkansc` and `vulkansc` but if another subset is added, like `vulkanes`, then some support values in the XML might be e.g. `vulkansc,vulkanes` and may want that to be excluded.

Instead of the excluding list `_remove_supported`, can this instead have a list of supported API subsets, like `_supported_subsets=['vulkan']` and then check that at least one of them is in `supported`?

Reply:
fixed it